app/controllers/concerns/state_file/state_file_controller_concern.rb

```diff
@@ -1,52 +1,27 @@
+# TODO i left these things in here because this concern exists but the thing is the concern is only included once (in the state file questions controller) so should we just move all this over to the controller itself?
 module StateFile
   module StateFileControllerConcern
     extend ActiveSupport::Concern
 
     included do
-      helper_method(
-        :current_tax_year, :filer_count, :state_name, :state_abbr, :ny?, :az?, :state_param
-      )
+      helper_method :current_tax_year, :state_name, :state_code
     end
 
-    private
-
-    def state_name
-      States.name_for_key(state_abbr)
-    end
-
-    def state_abbr
-      state_param&.upcase
-    end
-
-    def state_param
-      params[:us_state]
-    end
-
-    def ny?
-      state_param == "ny"
-    end
-
-    def az?
-      state_param == "az"
-    end
-
-    def service_type
-      case state_param
-      when "az" then :statefile_az
-      when "ny" then :statefile_ny
+    # TODO idea: state_code could prioritize getting the code from the current intake and if there isn't one, default to the params
+    def state_code
+      state_from_params = params[:us_state]
+      unless StateFile::StateInformationService.active_states.include?(state_from_params)
+        raise StandardError, state_from_params
       end
+      state_from_params
     end
 
-    def tenant_service
-      MultiTenantService.new(service_type)
+    def state_name
+      StateFile::StateInformationService.state_name(state_code)
     end
 
     def current_tax_year
-      tenant_service.current_tax_year
-    end
-
-    def filer_count
-      current_intake&.filer_count
+      MultiTenantService.new(:statefile).current_tax_year
```

Comment:
i tried deleting this initially but a test failed. i would like for this to look different in the future but fixing it felt out of scope